source/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -57,3 +57,51 @@ todosController.get('remaining');
 
 Note that `@each` only works one level deep. You cannot use nested forms like
 `todos.@each.owner.name` or `todos.@each.owner.@each.name`.
+
+Sometimes, you may only care if the items of an array have changed, rather than their specific properties.
```

Comment:
"you may only care if the items of an array have changed" is a bit ambigious IMO